assignment3/assignment_test.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:test/test.dart';
+import 'functions.dart';
+import 'graph.dart';
+
+void main() {
+  test('i want to test graph Class', () {
+    Graph testGraph = Graph(nodeId: 1, nodeName: "A");
+
+    String result = testGraph.nodeName;
+
+    expect(result, 'A');
+  });
+
+  test('i want to test Immediate parents  user', () {
```

Comment:
Add more test cases. You should aim to cover at least 80% of your code through these test cases.

Reply:
i have tried to write most of the test case that can be tested . i will keep that in mind and try to write functions that can be tested